sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -58,6 +58,12 @@ def get_lane_speed_key(physical_port, port_speed, lane_count):
             host_electrical_interface_id = appl_adv_dict[app_id].get('host_electrical_interface_id')
             if host_electrical_interface_id:
                 lane_speed_key = LANE_SPEED_KEY_PREFIX + host_electrical_interface_id.split()[0]
+    else:
+        speed_per_lane = int(port_speed) / 1000 / lane_count
+        speed_per_lane_int = int(speed_per_lane)
+        if speed_per_lane - speed_per_lane_int == 0:
+            speed_per_lane = speed_per_lane_int
+        lane_speed_key = LANE_SPEED_KEY_PREFIX + str(speed_per_lane) + 'G:' + str(lane_count)
```

Comment:
@Junchao-Mellanox why we need lookup for number of lanes? do you have sample media_setting.json with this for non-CMIS cable? Is this key with no of lanes captured in any HLD?